src/_posts/2017-11-16-katas-for-calisthenics.md

```diff
@@ -63,7 +63,7 @@ The easiest kata in this post (on its original form), but still difficult enough
 
 ### Why
 
-Of the katas on this post is the one on which more concepts are present. Which makes it interesting when comparing the solution with OOP. Use of the  is explicit with the changes introduced to the rules.
+Of the katas on this post is the one on which more concepts are present. Which makes it interesting when comparing the solution with OOP. Use of the [side effects at the boundaries]({% post_url 2017-10-12-functional-calisthenics %}#sideeffectsattheboundaries) is explicit with the changes introduced to the rules.
```

Comment:
These don't work- go to http://codurance-site-pr.s3-website-eu-west-1.amazonaws.com/site-restructuring/2017/11/16/katas-for-functional-calisthenics/ and click the "side effects at the boundaries" or "functional calisthenics" links, or any of the links under "Why". They 404.

Reply:
This is only showing "outdated" because the file was renamed

Reply:
This link doesn't work because the PR folder is not added to the links.
Also we know that on the functional calisthenics post, the local links with anchors doesn't work.
But we want to get this rolling, and fix that in a different PR

Reply:
Ahh, I see. After applying that to the path they seem to work.